source/uwp/Renderer/lib/AdaptiveTextBlockRenderer.cpp

```diff
@@ -35,6 +35,18 @@ namespace AdaptiveNamespace
         ComPtr<IAdaptiveTextBlock> adaptiveTextBlock;
         RETURN_IF_FAILED(cardElement.As(&adaptiveTextBlock));
 
+        ComPtr<IAdaptiveTextElement> textBlockAsTextElement;
+        RETURN_IF_FAILED(adaptiveTextBlock.As(&textBlockAsTextElement));
+        HString text;
+        RETURN_IF_FAILED(textBlockAsTextElement->get_Text(text.ReleaseAndGetAddressOf()));
+
+        // If the text is null, return immediately without constructing a text block
+        if (text.Get() == nullptr)
```

Comment:
So then it's a go with this one for 3702? If so could you cherry pick it to release/1,2

Reply:
Cherry-picked as #4304